src/formio/validators/plugins.js

```diff
@@ -1,41 +1,34 @@
 import {post} from '../../api';
 
-const errorMessageMap = {
-  'kvk-kvkNumber': 'Invalid Kvk Number',
-  'kvk-rsin': 'Invalid RSIN',
-  'kvk-branchNumber': 'Invalid Branch Number',
-  'phonenumber-international': 'Invalid international phonenumber',
-  'phonenumber-nl': 'Invalid Dutch phonenumber',
-};
+export const pluginsAPIValidator = {
+  key: `validate.backendApi`,
+  message(component) {
+    return component.t(component.errorMessage('Invalid'), {
+      field: component.errorLabel,
+      data: component.data,
+    });
+  },
+  check(component, setting, value) {
+    if (!value) return true;
 
-const pluginAPIValidator = plugin => {
-  let defaultMsg = errorMessageMap[plugin];
-  // catches undefined too
-  if (defaultMsg == null) {
-    defaultMsg = 'Invalid';
-  }
+    const plugins = component.component.validate.plugins;
+    const {baseUrl} = component.currentForm?.options || component.options;
 
-  return {
-    key: `validate.${plugin}`,
-    message(component) {
-      return component.t(component.errorMessage(defaultMsg), {
-        field: component.errorLabel,
-        data: component.data,
+    const promises = plugins.map(plugin => {
+      const url = `${baseUrl}validation/plugins/${plugin}`;
+      return post(url, {value}).then(response => {
+        const valid = response.data.isValid;
+        return valid ? true : response.data.messages.join('<br>');
       });
-    },
-    check(component, setting, value) {
-      if (!value) return true;
+    });
+    return Promise.all(promises).then(results => {
+      const anyValid = results.some(result => result === true);
 
-      const {baseUrl} = component.currentForm?.options || component.options;
-      const url = `${baseUrl}validation/plugins/${plugin}`;
-      return post(url, {value})
-        .then(response => {
-          const valid = response.data.isValid;
-          return valid ? true : response.data.messages.join('<br>');
-        })
-        .catch(() => false);
-    },
-  };
+      if (anyValid) return true;
+
+      return results.join('<br>');
+    });
```

Comment:
```suggestion
    }).catch(() => false);
```
I think this was removed by accident?